### PR TITLE
Development

### PR DESCRIPTION
**Pull request by cornets28**
Pull request

### CODE REVIEW

**Comment by Ispirett** [comment]
Hey, man great work with this project you are doing a fantastic job :1st_place_medal: . There are just a few key things that Mircoverse requires before this project can be completed.

 ###  Top Story
- [ ] **Top Story**
 It is wrapped by a pair of opening and closing <section> HTML tags
 It contains a story wrapped by a pair of opening and closing <article> HTML tags and a width of 100%
 From top to bottom, the story has an image, a title, and a small paragraph.

 ###  More Stories
- [ ] **More Stories**
* [ ] It is wrapped by a pair of opening and closing `<section>` HTML tags
* [ ] It has a header with the text: `More Stories`

- [ ]  The **Opinion section**  should be wrapped in a ` <section/>` tag also.

- [ ]  The **Feature section**  should be wrapped in a ` <section/>` tag also.
      This section articles wrapped by a pair of opening and closing <article> HTML tags. User article tags instead of divs. 

 ###  Latest Articles
- [ ]  The **Latest Articles section**  should be wrapped in a ` <section/>` tag also.
- [ ] The articles inside should follow these guidelines.
- [ ] It has eight articles wrapped by a pair of opening and closing `<article>` HTML tags
- [ ] Each article has a `width` property with a value of `100%`
- [ ] Each article has an image with a `float` property set with a value of `left`
- [ ] Each article's image has a `width` property with a value of `50%`
- [ ] Each article has a title with a `float` property set with a value of `right`
- [ ] Each article's title has a `width` property with a value of `50%`

 ### In the Magazine
- [ ]  The **In the Magazine**  should be wrapped in a ` <section/>` tag also.
- [ ]   This section should also have seven articles wrapped by a pair of opening and closing `<article>` HTML tags instead of divs

  
### Editor's picks

- [ ]  The **In the Magazine**  should be wrapped in a ` <section/>` tag also.
- [ ]   This section should also have seven articles wrapped by a pair of opening and closing `<article>` HTML tags instead of divs

 
###Featured Slideshow

- [ ]  The **In the Magazine**  should be wrapped in a ` <section/>` tag also.
- [ ]   This section should also have seven articles wrapped by a pair of opening and closing `<article>` HTML tags instead of divs

### Responsiveness
- [ ] Im getting a big around 1180px and less
  
bootstrap

### Nav 
- [ ] the nav should look like this at **1000px**  and less
- [ ] You can use display:hidden to hide the old nav and display:block to show the new one.
bootstrap2 


## Important
- [ ] Also next time please send a pull request link like this `https://github.com/CornetS28/the-news-week/pull/1` and not `git@github.com:CornetS28/the-news-week.git`
- [ ] There are alot of html errors on this page use this tool and please correct them https://validator.w3.org/

- [ ] The 
When finished with these changes please submit for another code review you can you this same pull request link.

**Comment by mcrd25** [comment]
# Currently under review by TSE

**Comment by samgaco** [comment]
Hi @CornetS28 :hand: 

Good job implementing the changes pointed out in the previous review. The project looks truly great but there are still some blemishes;

* When going to smallest screen size, you can see a horizontal scrolling bar. It shouldn't be there. The navigation bar is causing it.

* After the footer, there is some extra space that it shouldn't be there either.

* The containers should have a white background, mine while the vertical borders will preserve that WhiteSmoke color.

* You should correct the errors shown in the index file by https://validator.w3.org

Please :pray: correct these issues and submit back for another code review :muscle:  Regards!

**Comment by samgaco** [comment]
### Hi @SamuelC28 :hand:  Good job so far :100:  Some things to address before proceeding:


* The containers should have a white background, mine while the vertical borders will preserve that WhiteSmoke color. (The major part of the website is still showing a WhiteSmoke background)

* The navigation bar is still showing in the small screen sizes:

> ### Nav 
> - [ ] the nav should look like this at **1000px**  and less
> - [ ] You can use display:hidden to hide the old nav and display:block to show the new one.
> bootstrap2 

* There are many titles without the Newsweek logo (In the magazine, Editor's pick, Featured slideshows...) all these section titles should have a logo of Newsweek at the left, as it shows in the original page.

> crop2

* **In the magazine** section is not well implemented, compare:

### Your site:
Alt Text

### Original
Alt Text


* There is overlapping in some parts of the small screen version:

> overlap

### Please :pray: work on these problems and submit back for another code review. Good spirit and happy coding! :muscle:

**Comment by samgaco** [comment]
### Also, the branch has conflicts. Solve this issue before submitting back for a code review. :pray:

**Comment by bolah2009** [comment]
Currently under review

File: index.html [hunk 1]
```diff
@@ -2,64 +2,1083 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1, user-scalable=no">
+    <meta name="" content="width=device-width, initial-scale=1.0, maximum-scale=1, user-scalable=no">
```
Comment (bolah2009) [inline index.html]: `SUGGESTION`

Attribute `name` must not be empty. Learn more about `meta` tags [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#attr-name). Consider deleting this line or add `name="viewport"`.

File: index.html [hunk 2]
```diff
@@ -2,64 +2,1083 @@
...
+                          <li class="groupItem">
+                            <img src="./assets/images/smallimage1.jpg"  alt="">
+                            <a id="title" href="#">'RHOBH' Season 9 Cast Choose Between Kyle and Lisa</a>
+                          </li>
+                      </article>
+
+                      <article>
+                          <li class="groupItem">
+                            <img src="./assets/images/smallimage1.jpg"  alt="">
+                            <a id="title" href="#">'RHOBH' Season 9 Cast Choose Between Kyle and Lisa</a>
```
Comment (bolah2009) [inline index.html:348]: You seem to be using multiple the same `id` in multiple places. Consider making these `id` unique across other elements.

File: index.html [hunk 3]
```diff
@@ -2,64 +2,1083 @@
...
```
Comment (bolah2009) [inline index.html:355]: You seem to be using multiple the same `id` in multiple places. Consider making these `id` unique across other elements.

**Review by bolah2009 — APPROVED** [review]
## Nice Job [Sammy](../../) :clap:

The [clone](https://raw.githack.com/CornetS28/the-news-week/development/index.html) looks good to me and is responsive except for the small screen (see screenshot below). :+1: 

current newsweek com

#### You can recreate this issue (Image above) by using the console on chrome (Inspect `Ctrl+Shift+I`) and the Toggle device toolbar `Ctrl+Shift+M`.

#### Check out the comments below and [these errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FCornetS28%2Fthe-news-week%2Fdevelopment%2Findex.html) on using an heading (h1-h6) element in an `<article>` and `<section>` elelment. I strongly recommend making these chages before merging.

Good luck with your next project. :star2: 

Cheers
[Bola](https://bolabuari.com/)

**Pull request merged**